frontend/src/Components/ReusableComponents/ReusableFunctions.js

```diff
@@ -1,9 +1,31 @@
 import password from "secure-random-password";
 
+export const passwordRegEx = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z\d]{8,}$/;
 export const pwSuggestion = (length, func1, func2) => {
    let suggestedPw = password.randomPassword({ length, characters: [password.lower, password.upper, password.digits] });
    func1(suggestedPw);
    if (func2) {
       func2(suggestedPw);
    }
 };
+
+export const redirectTo = (func, path, delay) => {
+   setTimeout(() => {
+      func(path);
+   }, delay);
+};
+
+export const PasswordRequirements = (password, confPassword) => {
```

Comment:
Great extracting this one, I would suggest maybe create it as a separate component in separate file.

Reply:
I will do it, thanks.